### DISCUSSION TITLE
Tinker pdbxyz does not take key file into account

### DISCUSSION THREAD

**Discussion by lukaszmilewski** [TinkerTools/tinker-gpu#188] [Q&A] [answered]
Hi,

i have an issue with pdbxyz tool.

Tinker pdbxyz does not take key file into account. 

I am using the command:
`pdbxyz my-file.pdb -key tinker.key`

tinker.key file inside:
#forcefield
parameters /home/users/lmilewski/programs/force_field/charmm27.prm

The output is the same as default without key file (not this force field). Atom types are not corresponding with charmm27 force field.
What can I do to use force field which is not the default one?

Thanks for your help!

**Comment by zhi-wang** [comment]
Hi, I will move this issue to https://github.com/TinkerTools/tinker/issues

**Comment by zhi-wang** [comment, marked as answer]
Here is the link where Prof. Ren and Prof. Ponder answered your question. https://github.com/TinkerTools/tinker/issues/103
